test/tools/OpenCover/OpenCover.psm1

```diff
@@ -555,7 +555,7 @@ function CreateOpenCoverCmdline($target, $outputLog, $targetArgs)
         "-hideskipped:all",
         "-mergeoutput",
         "-filter:`"+[*]* -[Microsoft.PowerShell.PSReadLine]*`"",
-        "-targetargs:`"-EncodedCommand $base64targetArgs`""
+        "-targetargs:`"-NoProfile -EncodedCommand $base64targetArgs`""
```

Comment:
this is fine, and another possible parameter to use is `-noninteractive` which will keep PSReadLine from being loaded.